llvm/test/CodeGen/PowerPC/inline-asm-clobber-warning.ll

```diff
@@ -1,7 +1,7 @@
 ; RUN: llc < %s -verify-machineinstrs -mtriple=powerpc-unknown-unkown \
-; RUN:   -mcpu=pwr7 2>&1 | FileCheck %s
```

Comment:
Why change to `O0`?

Reply:
The purpose is to avoid optimizing the clobbering of r30 and r31 in @test_r31_r30_clobber().